README.md

```diff
@@ -74,11 +74,18 @@ Valhalla uses the [GNU Build System](http://www.gnu.org/software/automake/manual
     #if you plan to compile with python bindings, see below for more info
     sudo apt-get install -y python-all-dev
 
+To install on macOS, you need to install its dependencies with [Homebrew](http://brew.sh):
+
+    brew install autoconf automake libtool protobuf-c boost-python libspatialite pkg-config lua czmq
```

Comment:
this seems to be missing stuff for the prime_server bits specifically. im sure you can just get and compile them but maybe its worth it at this point to get prime_server into brew or valhalla even? a quick google seemed to make it look pretty easy actually (docker-esque)